### PR TITLE
Make the status Labels translatable

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- First draft of mechanism for statuses translations
- Cover: ApiRules, Nodes (cluster overview), Functions

**Related issue(s)**

### CODE REVIEW

File: core-ui/src/components/ApiRules/ApiRuleStatus/ApiRuleStatus.js [hunk 1]
```diff
@@ -22,13 +23,20 @@ const resolveAPIRuleStatus = statusCode => {
...
   if (!apiRule.status?.APIRuleStatus) {
     return null;
   }
 
   const { code, desc } = apiRule.status.APIRuleStatus;
   return (
-    <StatusBadge type={resolveAPIRuleStatus(code)} tooltipContent={desc}>
+    <StatusBadge
+      i18n={i18n}
+      resourceKind={'api-rules'}
```
Comment (qbalukom) [inline core-ui/src/components/ApiRules/ApiRuleStatus/ApiRuleStatus.js:36]: wouldn't resourceKind='api-rules' be enough here?

File: core-ui/src/components/Clusters/views/ClusterStorageType.js [hunk 2]
```diff
@@ -3,36 +3,18 @@ import { useTranslation } from 'react-i18next';
...
-    tooltip: t('clusters.storage.descriptions.unknown'),
-  };
+  const knownStorage = ['localStorage', 'sessionStorage', 'inMemory'];
 
   return (
     <StatusBadge
-      type={descriptions[storage] ? 'info' : 'warning'}
-      tooltipContent={description.tooltip}
+      i18n={i18n}
+      resourceKind={'clusters'}
```
Comment (qbalukom) [inline core-ui/src/components/Clusters/views/ClusterStorageType.js:14]: wouldn't resourceKind='clusters' be enough here?

File: core/src/i18n/en.yaml [hunk 3]
```diff
@@ -1,3 +1,4 @@
+aa: s
```
Comment (qbalukom) [inline core/src/i18n/en.yaml:1]: ???

**Review by qbalukom — CHANGES_REQUESTED** [review]

**Review by qbalukom — APPROVED** [review]

File: core/src/i18n/en.yaml [hunk 4]
```diff
@@ -74,6 +74,10 @@ api-rules:
     loading: Loading services...
     ns-no-services: No Services in this Namespace.
     placeholder: Choose the service to expose.
+  statuses:
+    error: Error
+    ok: Ok
```
Comment (majakurcius) [inline core/src/i18n/en.yaml:79]:
```suggestion
    ok: OK
```

**Review by majakurcius — DISMISSED** [review]

**Review by qbalukom — APPROVED** [review]

**Review by majakurcius — APPROVED** [review]

**Pull request merged**